Makefile

```diff
@@ -115,3 +115,31 @@ report-celery-beat:
 
 admin-user:
 	superset fab create-admin
+
+# Create a postgres instance using docker
```

Comment:
@fisjac could you specify the reason to add this new docker image in this PR?

Reply:
@justinpark I'm going to close this PR and reopen another on a different up-stream branch